dynatrace-aws-log-forwarder-template.yaml

```diff
@@ -746,10 +746,14 @@ Outputs:
   FirehoseArn:
     Description: "Firehose ARN"
     Value: !GetAtt FirehoseLogStreams.Arn
+    Export:
+      Name: !Join [ ":", [ !Ref "AWS::StackName", FirehoseARN ] ]
 
   CloudWatchLogsRoleArn:
     Description: "CloudWatch Logs role ARN allowing streaming to Firehose"
     Value: !GetAtt CloudWatchLogsRole.Arn
+    Export:
+      Name: !Join [ ":", [ !Ref "AWS::StackName", CloudWatchARN ] ]
```

Comment:
Could you tell me more about your use case? The way I understand it, you are using `deploy` option, but use something else than our `subscribe` for subscribing, and would like to get those outputs available for use there (i.e. some other CloudFormation stack that deploys the actual resources that produce logs?), without actually querying cli for those arns the way `subscribe` does it.

Reply:
@dt-be the use-case we have for an export name is for codifying the output values in our serverless config. Rather than running the script to subscribe the LogGroups we can add the `serverless-plugin-log-subscription` plugin to our serverless lambda that can import the Cloudwatch and Firehouse ARNs directly from the stack and auto create a LogSubscription to attach to the lambda LogGroups

Example serverless.yml code
```yaml
service: my-service

plugins:
  - serverless-plugin-log-subscription

custom:
  logSubscription:
    - enabled: true
      destinationArn: 'Fn::ImportValue': 'dynatrace-aws-logs-FirehouseARN'
      roleArn: 'Fn::ImportValue': 'dynatrace-aws-logs-CloudWatchARN'
```